src/basic/src/ScalarSequence.C

```diff
@@ -1729,14 +1729,14 @@ ScalarSequence<T>::subBasicHistogram(
   for (unsigned int j = 0; j < dataSize; ++j) {
     double value = m_seq[j];
     if (value < minHorizontalValue) {
-      bins[0] += value;
+      bins[0]++;
```

Comment:
Same bug but different functions.

`value` is the sample value.  For a histogram you don't add the value of the sample to the bin height, you add 1 to it if the sample is in that bin.  I'll update the commit message.